internal/server/v0/statpoint/golden/get_stat_value/umemployed.json

```diff
@@ -1,3 +1,3 @@
 {
-  "value": 6270000
+  "value": 5952000
```

Comment:
Likely from BLS refresh.  (Might be nice to tell from the path of the file, or some other way what the place is)